### PR TITLE
DOCSP-10482: rearrange CRUD landing page content

### PR DESCRIPTION
**Pull request by ccho-mongodb**
JIRA:
https://jira.mongodb.org/browse/DOCSP-10482

Staging:
[Main CRUD page](https://docs-mongodborg-staging.corp.mongodb.com/9c87efc/node/docsworker/DOCSP-10482-crud-landing-reorg/fundamentals/crud)

[Retrieve CRUD page](https://docs-mongodborg-staging.corp.mongodb.com/9c87efc/node/docsworker/DOCSP-10482-crud-landing-reorg/fundamentals/crud/read-operations/retrieve)

[Insert CRUD page](https://docs-mongodborg-staging.corp.mongodb.com/9c87efc/node/docsworker/DOCSP-10482-crud-landing-reorg/fundamentals/crud/write-operations/insert)

[Delete CRUD page](https://docs-mongodborg-staging.corp.mongodb.com/9c87efc/node/docsworker/DOCSP-10482-crud-landing-reorg/fundamentals/crud/write-operations/delete)

### CODE REVIEW

File: source/fundamentals/crud/write-operations/insertDelete.txt [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+to the ``insertMany()`` method as shown below:
+
+.. code-block:: javascript
+
+   pizzaCollection.insertMany(pizzaDocuments);
+
+For a runnable example, see the :doc:`insertMany() </usage-examples/insertMany>`
+usage example.
+
+Delete
```
Comment (terakilobyte) [inline source/fundamentals/crud/write-operations/insertDelete.txt:71]: It feels odd reading this to have insert and delete operations on the same page, then have update operations on the next page in the toc. Maybe break them into their own pages?

**Comment by nraboy** [comment]
For the "Retrieve Data" page, I'd like to see example output for each of the three example used (find, aggregate, watch).

**Comment by nraboy** [comment]
For the "Insert a Document" page, I'd like to see example output for the insertOne and insertMany operations.

**Comment by nraboy** [comment]
Sample output would be appreciated for the "Delete a Document" example as well.

**Pull request merged**